jobs/views.py

```diff
@@ -5,20 +5,23 @@
 from django.views.generic.edit import CreateView
 from django.core.urlresolvers import reverse_lazy
 
-from .models import Job, Company
+from .models import Job, Meetup, Company
 from .forms import JobForm
 
 
 def jobs(request):
-    job_offers = Job.objects.filter(ready_to_publish=True).filter(expiration_date__gte=timezone.now())
+    job_offers = Job.objects.filter(ready_to_publish=True, expiration_date__gte=timezone.now())
+    meetup_list = Meetup.objects.filter(ready_to_publish=True, expiration_date__gte=timezone.now())
```

Comment:
ready_to_publish doesn't necessarily mean it's published, does it?  We may want to filter based on whether there is a published_date as well (not instead, since we could have something that's been published and then removed)